app/services/scene-collections/nodes/overlays/scenes.ts

```diff
@@ -26,14 +28,24 @@ export class ScenesNode extends ArrayNode<ISchema, IContext, Scene> {
 
     return {
       name: scene.name,
+      sceneId: scene.getSource().sourceId,
```

Comment:
is `sourceId` the same as `sceneId`? it seems strange to me that we would need to call `getSource` to grab the sceneId of a scene instance